src/main/java/com/taskadapter/redmineapi/internal/URIConfigurator.java

```diff
@@ -110,7 +110,11 @@ private URI createURI(String query,
                 builder.addParameter("key", apiAccessKey);
             }
             if (!query.isEmpty()) {
-                builder.setPath(builder.getPath() + "/" + query);
+                String builderPath = builder.getPath();
+                if (null == builderPath) {
+                    builderPath = "";
+                }
+                builder.setPath(builderPath + "/" + query);
```

Comment:
how about just
     builder.getPath() == null? "" : builder.getPath()

Reply:
OK! It's better.

Reply:
I have modified. @alexeyOnGitHub